apisix/admin/global_rules.lua

```diff
@@ -150,14 +150,14 @@ function _M.patch(id, conf, sub_path)
         if code then
             return code, err
         end
+        utils.inject_timestamp(node_value, nil, true)
```

Comment:
The laste args is nil

Reply:
The seme as below， only upstream is right

Reply:
No. The placeholder is true. See other changed files.